connector/mqtt/vernemq/broker/bin/vmq_dojot/vmq_dojot.sh

```diff
@@ -45,12 +45,12 @@ _connectEJBCA()
   START_TIME=$(date +'%s')
   echo "Waiting for dojot EJBCA Broker fully start. Host ${EJBCA_ADDRESS}..."
   echo "Try to connect to dojot EJBCA Broker ... "
-  RESPONSE=$(curl --fail -s "${certEjbcaApiUrl}"/ejbca/version || echo "")
+  RESPONSE=$(curl --fail -s "${certEjbcaApiUrl}"/healthcheck || echo "" | jq '.status')
   echo "$RESPONSE"
-  while [ -z "${RESPONSE}" ]; do
+  while [ -z "${RESPONSE}" ] || [ "${RESPONSE}" = '"ok"' ]; do
```

Comment:
Why retry to connect if the `${RESPONSE}` was OK in the previous instruction ?

Reply:
You are right, it should be a "different" operator, very well observed, thanks!